### ISSUE TITLE
xml2js-xpath doesn't handle XML tags (nodes) with a hyphen correctly

### ISSUE THREAD

**Issue by nmilosevic** [dsummersl/node-xml2js-xpath#11]
Hi,

It seems that the package is not handling tags (nodes) with a hyphen correctly. I run a simple node app with the following code:
```
let xml2js = require('xml2js');
let xpath = require('xml2js-xpath');

let parser = new xml2js.Parser();
let xml = '<test><tag-1>test1</tag-1><tag2>test2</tag2></test>';

parser.parseString(xml, function (error, result) {
    if (!error) {
        try {
            console.log(xpath.find(result, '/test/tag-1'));
        } catch (err) {
            console.error(err);
        }
    } else {
        console.error(error);
    }
});
```

The program output is [].
If I change '/test/tag-1' into '/test/tag2', program output is [ 'test2' ].

**Comment by dsummersl** [comment]
Thanks for the ticket @nmilosevic  - I'll try to take a crack at this this weekend.

**Comment by dsummersl** [comment]
Thanks for the clear test case, I've added support for this and released a new version 0.9.0 - I realize I had been using a fairly naive interpretation of what it meant to be an XML tag name. It should be a bit better.
